includes/Assets.php

```diff
@@ -501,8 +501,6 @@ public function enqueue_front_scripts() {
             }
         }
 
-        wp_enqueue_script( 'dokan-login-form-popup' );
```

Comment:
this is used in 'report abuse' and 'follow store' in dokan pro as dependency of 'dokan-report-abuse' and 'dokan-follow-store' js scripts

Reply:
So, deleting it may arise issues. Please keep this line as it was.

Reply:
that will enqueue it in all the vendor dashboard pages.

Reply:
Yes, that's why it's added. Like, if any vendor's Login time is exceeded, then this popup will be shown. If you delete these enqueue this would not work.

Reply:
ok vai. got it.